/stereo_aruco_detector:
  ros__parameters:
    R: [
      0.9999836097161154, -0.0012990731712661104, 0.0055760835739236524,
      0.0013090511868552913, 0.9999975480160502, -0.0017861542143542514,
      -0.0055737495564366114, 0.0017934243176000798, 0.999982858325631
    ]
    t: [0.119782, -0.000207959, 0.000447725]
    T_global: [
      -1.000, 0.001, 0.020, 0.020,
      -0.020,-0.004,-1.000,-0.111,
      -0.001,-1.000, 0.004, 0.038,
       0.000, 0.000, 0.000, 1.000,
    ]
    aruco_dict: DICT_5X5_250
    aruco_ids:
    - 51
    - 52
    - 53
    - 54
    - 55
    - 56
    - 57
    - 58
    - 59
    - 60
    - 61
    - 62
    - 63
    - 64
    - 65
    intrinsics_left: [535.235, 535.51, 642.675, 354.7465]
    intrinsics_right: [533.36, 533.705, 643.02, 362.2895]
    dist_left: [-0.0563235, 0.0313255, 0.00019802, -0.000175828, -0.012571]
    dist_right: [-0.0526495, 0.0235335, 0.000350472, 0.000447492, -0.00913548]
    stereo_left_topic: /zed/left/image_raw
    stereo_right_topic: /zed/right/image_raw
    use_sim_time: false
    visualize: true
